vae/config/packages.yaml

```diff
@@ -0,0 +1,5 @@
+packages:
+  all:
+    providers:
+      tbb: [fairsoft_backports.intel-tbb]
+      blas: [fairsoft_backports.openblas]
```

Comment:
As you added `vae/config` in 81e0e6a14f925dba17c2c8265667dd77b17191ae you should modify `vae/thisvae.sh` in that same commit too:
```diff
-fairsoft_configdir=../../../../config
+fairsoft_configdir=../../../config
```

Reply:
done